kornia/geometry/boxes.py

```diff
@@ -121,6 +207,7 @@ def __init__(self, boxes: torch.Tensor, raise_if_not_floating_point: bool = True
             raise ValueError(f"Boxes shape must be (N, 4, 2) or (B, N, 4, 2). Got {boxes.shape}.")
 
         self._boxes = boxes
```

Comment:
```suggestion
        self._data = boxes
```
Small suggestions (maybe for a later PR)
can we rename this to data -- seems more natural later when we do `boxes._data` than `boxes._boxes`. Besides, a thought that maybe a more appropriate name for the class would be in singular `Box` which can be less confusing when having just one instance o a box. Later we can have special cases with inheritance like `BoundingBox`, `RotatedBox`n, etc. Also wondering if makes sense to inherit from Tensor as well.

Reply:
One concern about inherit from tensor is that we will need to consider many overrides. Like how to define the plus of two boxes? `Box(A) + Box(B) = Box([A, B])`, etc.